tests/core/test_document.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+from ragna.core import DocxDocumentHandler, LocalDocument
+
+
+@pytest.fixture()
```

Comment:
If we only have a single test right now, I don't think we need these fixtures. We can just inline them.